### PR TITLE
Update credential-guard-requirements.md

### PR DESCRIPTION
**Pull request by illfated**
~~Ref. closed issue ticket #4025 and PR #8435~~

~~Just to make it clear that Credential Guard is not supported on Windows 10 Pro or Windows 10 Home edition~~

Thanks to @JonZeolla for raising the question by opening the ticket.
Thanks to @tecxx for taking the time to create and follow up CRM:0773000358 (MSRC).

- MarkDown indent marker compatibility whitespace (single space, 3 occurrences)

The question about **Credential Guard: Enterprise & Education SKU** will be handled in PR #8435 instead of this one.

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
Single line change in the **Applies to** section.

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
@illfated Can you resolve the merge conflicts in the meantime that we escalate this to author(s)? Thank you.

**Comment by illfated** [comment]
Sure thing. Thank you for posting and mentioning the issue. Will fix.

File: windows/security/identity-protection/credential-guard/credential-guard-requirements.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ ms.reviewer:
 
 # Windows Defender Credential Guard: Requirements
 
+
```
Comment (illfated) [inline windows/security/identity-protection/credential-guard/credential-guard-requirements.md:21]: Remove unneeded blank line:
```suggestion
```

File: windows/security/identity-protection/credential-guard/credential-guard-requirements.md [hunk 2]
```diff
@@ -58,11 +58,12 @@ For information about Windows Defender Remote Credential Guard hardware and soft
 
 When Windows Defender Credential Guard is enabled, specific authentication capabilities are blocked, so applications that require such capabilities will break. Applications should be tested prior to deployment to ensure compatibility with the reduced functionality.
 
->[!WARNING]
+
```
Comment (illfated) [inline windows/security/identity-protection/credential-guard/credential-guard-requirements.md:61]: Remove unneeded blank line:
```suggestion
```

**Comment by illfated** [comment]
I have reduced this PR to only handle missing MarkDown indent marker compatibility spacing.
Please use PR #8435 to correct the page title section where the question about Windows 10 versions is handled.

**Comment by e0i** [comment]
Forgot to call for copy editing.

cc @JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Conclusion: 3 whitespace edits 
Proof: **Hide whitespace changes:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8436/files?diff=unified&w=1 
-- result: Whitespace-only changes -- no content shown.

**Review by DulceMontemayor — APPROVED** [review]

**Pull request merged**